### ISSUE TITLE
[Hunter] Piercing Shots + Mangle

### ISSUE THREAD

**Issue by Elaatu** [SunwellTracker/issues#256]
**Opis:** 
Debuff Mangle który zwiększa dmg bleedów o 30% nie działa na Piercing Shots.

**Logi bez Mangla:** http://worldoflogs.com/reports/p2t61xxz75dz5eax/log/

[22:00:31.268] Valcy Piercing Shots Heroic Training Dummy 145
[22:00:32.241] Valcy Piercing Shots Heroic Training Dummy 145
[22:00:33.229] Valcy Piercing Shots Heroic Training Dummy 145
[22:00:34.213] Valcy Piercing Shots Heroic Training Dummy 145
[22:00:35.261] Valcy Piercing Shots Heroic Training Dummy 145
[22:00:36.275] Valcy Piercing Shots Heroic Training Dummy 145
[22:00:37.236] Valcy Piercing Shots Heroic Training Dummy 145

**Logi z Manglem:** http://worldoflogs.com/reports/vwnrhlu2uv0ulnd1/log/
[22:04:50.249] Heroic Training Dummy's Mangle (Cat) is refreshed by Bronk
[22:04:51.253] Valcy Piercing Shots Heroic Training Dummy 145
[22:04:52.186] Valcy Piercing Shots Heroic Training Dummy 145
[22:04:53.224] Valcy Piercing Shots Heroic Training Dummy 145
[22:04:54.169] Valcy Piercing Shots Heroic Training Dummy 145

**Query:** [{"spellNames": ["Piercing Shots"]}, {"spellNames": ["Mangle (Cat)"]}]

**Jak działa na sunwellu:** 
Debuff Mangle  ( **All bleed** effects cause 30% additional damage ) nie wpływa na dmg bleeda huntera MM Piercing Shots

**Jak powinno działać:** 
Debuff Mangle powinien zwiększać o 30% dmg z Piercing Shots

**Dowód:** 

Opis Piercing Shots - Your critical Aimed, Steady and Chimera Shots cause the target to **bleed** for 30% of the damage dealt over 8 sec.

Opis  Debuff Mangle - **All bleed** effects cause 30% additional damage

http://www.warcrafthuntersunion.com/2010/11/how-piercing-shots-works/

The 30% Bleed Buff
It’s worth noting that Piercing Shots is a bleed effect, and its damage is increased by the 30% bleed debuff, which can be provided by the hunter pets hyena, boar, and rhino (exotic). The buff is also provided by arms warriors, subtlety rogues, and feral druids.

**Comment by dathremar** [comment]
podbijam reporta.
Od siebie dodam kolejny bug zwiazany z tym talentem, otoz:
DMG piercing shotsow buguje sie, jezeli jakas porcja DMG procujacego talent spella (chimera/aimed/steady) zostala zabsorbowana. Jezeli DMG zostanie calkowicie zabsorbowany, to piercing shotsy tickna 8 razy jako miss, jezeli zabsorbuje sie np. 50% dmg, to piercing  shotsy zadadza DMG rowny 30% z tych niezabsorbowanych DMG. 
Tymczasem DMG zawsze powinien byc rowny 0.3*(DMG zadany+dmg zabsorbowany)
tldr. absorby chimery/aimeda/steady shota  nie powinny miec zadnego wplywu na dmg piercing shotsow

https://forum.sunwell.pl/index.php?/topic/7925-krytyczny-efekty-on-crit-vs-absorb/&do=findComment&comment=70030   wszystkie dowody sa zawarte w pierwszym poscie w zalinkowanym temacie.

**Comment by Riztazz** [comment]
Resolved with next realm update
